app/src/lib/WalletButton.svelte

```diff
@@ -0,0 +1,23 @@
+<button
+	class="wallet-adapter-button"
+	class:justify-between={$$slots['start-icon'] || $$slots['end-icon']}
```

Comment:
$$slots are accepted by Svelte, but not recommended (performance issues). Since all the values that this component needs are coming from `$walletStore`, we don't need to use <slot />.